### PR TITLE
[DOCS-4858] Add ordinality info and update images to Log Detection doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

- Add information about the new ordinality feature for rule cases
- Updates images
- Resizes images

### Motivation

[DOCS-4858](https://datadoghq.atlassian.net/browse/DOCS-4858)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-4858]: https://datadoghq.atlassian.net/browse/DOCS-4858?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/security/cloud_siem/log_detection_rules.md [hunk 1]
```diff
@@ -176,9 +176,11 @@ Click the **Advanced** option to add queries that will **Only trigger a signal w
 
 ### Trigger
 
-{{< img src="security/security_monitoring/detection_rules/define_rule_case.png" alt="Define the rule case" >}}
+{{< img src="security/security_monitoring/detection_rules/define_rule_case_ordinality.png" alt="The set rule case section showing the default settings" style="width:80%;" >}}
 
-Rule cases, such as `a > 3`, are evaluated as case statements. Thus, the first case to match generates the signal. Click and drag your rule cases to manipulate their ordering.
+Enable ordinality rule cases if you want to use the `then` operator. An example of an ordinatlity rule case is where a signal is generated if query A occurs and then query B occurs.
```
Comment (nimishaaa) [inline content/en/security/cloud_siem/log_detection_rules.md:181]: As mentioned - let's remove the term "ordinality"

File: content/en/security/cloud_siem/log_detection_rules.md [hunk 2]
```diff
@@ -102,15 +102,15 @@ In this example, when greater than five failed logins and a successful login exi
...
+{{< img src="security/security_monitoring/detection_rules/new_term.png" alt="Define the search query" style="width:80%;" >}}
 
 Construct a search query using the same logic as a [log explorer search][1]. Each query has a label, which is a lowercase ASCII letter. The query name can be changed from an ASCII letter by clicking the pencil icon.
 
 **Note**: The query applies to all Datadog events and ingested logs which do not require indexing.
 
 #### Learned value
 
-{{< img src="security/security_monitoring/detection_rules/learning_duration.png" alt="Define the learned value" >}}
+{{< img src="security/security_monitoring/detection_rules/learning_duration.png" alt="Define the learned value" style="width:80%;" >}}
```
Comment (nimishaaa) [inline content/en/security/cloud_siem/log_detection_rules.md:113]: will need to update the screenshot with the update name!

**Comment by maycmlee** [comment]
Thanks @nimishaaa! Made the updates, let me know if it looks good and when you want to merge it on Monday.

File: content/en/security/cloud_siem/log_detection_rules.md [hunk 3]
```diff
@@ -176,9 +176,11 @@ Click the **Advanced** option to add queries that will **Only trigger a signal w
 
 ### Trigger
 
-{{< img src="security/security_monitoring/detection_rules/define_rule_case.png" alt="Define the rule case" >}}
+{{< img src="security/security_monitoring/detection_rules/define_rule_case2.png" alt="The set rule case section showing the default settings" style="width:80%;" >}}
 
-Rule cases, such as `a > 3`, are evaluated as case statements. Thus, the first case to match generates the signal. Click and drag your rule cases to manipulate their ordering.
+Enable **Create rules cases with the Then operator** if you want to generate a signal for the example, if query A occurs and then query B occurs.
```
Comment (nimishaaa) [inline content/en/security/cloud_siem/log_detection_rules.md:181]: There's a small limitation that we may want to call out as well - "THEN operator can only be used in one case"

Also the feature is now GA so we should be good to merge 🙇🏽‍♀️

Comment (maycmlee) [inline content/en/security/cloud_siem/log_detection_rules.md:181]:
```suggestion
Enable **Create rules cases with the Then operator** if you want to trigger a signal for the example: If query A occurs and then query B occurs. The Then operator can only be used on one rule case.
```

**Review by drichards-87 — APPROVED** [review]
Looks good! PR is approved.

**Pull request merged**